chromium_src/chrome/elevation_service/elevator.cc

```diff
@@ -3,13 +3,20 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at https://mozilla.org/MPL/2.0/. */
 
+#include "chrome/elevation_service/elevator.h"
+
+#include <windows.h>
+#include <winerror.h>
+
+#include <intsafe.h>
+
+#include "base/win/windows_types.h"
```

Comment:
@bridiver Is there are particular reason for these Windows headers to be listed as they are? I had to correct an issue in this file recently, and now `clang-format` for some insists in reordering these Windows inclusions to:

```c++
 #include "chrome/elevation_service/elevator.h"
 
 #include <windows.h>
-#include <winerror.h>
 
 #include <intsafe.h>
+#include <winerror.h>
 
 #include "base/path_service.h"
 #include "base/win/windows_types.h"
```